modules/porous_flow/doc/content/modules/porous_flow/multiapp_fracture_flow_diffusion.md

```diff
@@ -0,0 +1,142 @@
+
+\begin{equation}
+H = h(T_{f} - T_{m}) \ .
+\end{equation}
+
+In the MultiApp approach, $H$ is generated as an `AuxVariable` by the `fracture` App, so exists only in the `fracture` subdomain.  It is then passed to the `matrix` App, and applied as a `DiracKernel`.
+
+The boundary conditions are "no flow", except for the very left-hand side of the fracture domain, where temperature is fixed at $T_{f} = 1$.  The initial conditions are $T_{m} = 0 = T_{f}$.
+
+Each of the heat capacities are unity, $c_{m} = 1 = c_{f}$, the conductivity in the fracture is $\lambda_{f} = 1.0$, and is $\lambda_{m} = 10^{-3}$ in the matrix.  The fracure has aperture $a=10^{-2}$.
+
+Assume that $h_{s} = 0.02$.  This is the heat-transfer coefficient to use in the conforming case, since the matrix nodes align exactly with the fracture.  [eqn.suggested.h] may be used for the nonconforming case.  In this situation $L_{\mathrm{left}} = 0.1 = L_{\mathrm{right}}$ and $\lambda_{m}^{nn} = 10^{-3}$, so $h = 0.01$.
+
+Each simulation runs with `end_time = 100`.
+
+
+## No MultiApp: the benchmark
+
+In the conforming case, a MultiApp approach need not be taken, and the Kernels (given by [eqn.coupled.basic]) are:
+
+!listing fracture_diffusion/no_multiapp.i block=Kernels
+
+Evaluating the `fromFracture` heat transfer Kernel only on `block = fracture` implements the Dirac delta function $\delta(f)$ in [eqn.coupled.basic].  The matrix temperature is shown in [no_multiapp_matrix_T]
+
+!media porous_flow/examples/multiapp_flow/no_multiapp_matrix_T.png
+	style=width:60%;margin:auto;padding-top:2.5%;
+	id=no_multiapp_matrix_T
+	caption=Benchmark matrix temperature at the end of simulation
+
+The solution of this problem is [exactly solvable](https://github.com/idaholab/moose/blob/master/modules/misc/doc/fracture_flow) in terms of error functions, and by choosing appropriate time-step and element sizes, MOOSE produces the expected result.  Some examples of how MOOSE's solution depends upon time-step size are shown in [frac_no_multiapp_frac_T].  Evidently, reducing the time-step below 10.0 does not impact the solution very much.  Hence, the solution using $\Delta t = 0.125$ is used as the *benchmark* for the remainder of this section.
+
+!media porous_flow/examples/multiapp_flow/frac_no_multiapp_frac_T.png
+	style=width:60%;margin:auto;padding-top:2.5%;
+	id=frac_no_multiapp_frac_T
+	caption=Impact of time-step size on the fracture temperature at the end of the simulation
+
+## A MultiApp approach for the conforming case
+
+In this case, the matrix App is the main App, and the fracture App is the subApp.  The fracture input file has the following features.
+
+- An `AuxVariable` called `transferred_matrix_T` that is $T_{m}$ interpolated to the fracture mesh.
+
+- An `AuxVariable` called `joules_per_s` that is the heat rate coming from each node.  Mathematically this is $h(T_{f} - T_{m})L$, where $L$ is the "volume" modelled by the fracture node.  This is populated by the `save_in` feature:
```

Comment:
Why does save_in work in this case?  In the previous 1D case you said not to use save_in because it is scaled by the element volume, I think.

Reply:
True.  The `save_in` works in all cases except the previous one.  I've modified the discussion of the previous one to read:


(The inclusion of nodal volume is *exactly* what is required in the [diffusion](porous_flow/multiapp_fracture_flow_diffusion.md), [2D](porous_flow/multiapp_fracture_flow_PorousFlow_2D.md) and [3D](porous_flow/multiapp_fracture_flow_PorousFlow_3D.md) cases, since the amount of heat energy (measured in Joules) is transferred between the Apps in those cases, but in the current situation `CoupledForce` requires an energy density measured in Joules.m$^{-3}$.)